### PR TITLE
[develop] Update Hera modulefiles to allow the SRW to build and run once again.

### PR DESCRIPTION
**Pull request by MichaelLueken**
## DESCRIPTION OF CHANGES: 
The HPC-stack on Hera was updated at 17:05 today, October 13.  This update caused the SRW to fail to build due to a new dependency on the nccmp module.  This update will correct the issue and allow the SRW to once again build and run on Hera.

### Type of change
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## TESTS CONDUCTED: 
Following the modifications, the SRW was successfully built on Hera and the fundamental WE2E tests were submitted.  The tests are still running as of this time.

- [X] hera.intel - fundamental WE2E tests are still running at this time
- [ ] orion.intel
- [ ] cheyenne.intel
- [ ] cheyenne.gnu
- [ ] gaea.intel
- [ ] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins
- [X] fundamental test suite
- [ ] comprehensive tests (specify *which* if a subset was used)

## CHECKLIST

- [X] My code follows the style guidelines in the Contributor's Guide
- [X] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [X] My changes do not require updates to the documentation (explain).
- [X] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

**Review by danielabdi-noaa — COMMENTED** [review]
@MichaelLueken I have made similar changes to jet and orion build modulefiles too thinking we would be encountering same issue as Hera. Now it looks like the opposite is happening, so you may want to move back nccmp to where they were before in jet/orion build modulefiles.

**Review by danielabdi-noaa — APPROVED** [review]

**Comment by MichaelLueken** [comment]
> @MichaelLueken I have made similar changes to jet and orion build modulefiles too thinking we would be encountering same issue as Hera. Now it looks like the opposite is happening, so you may want to move back nccmp to where they were before in jet/orion build modulefiles.

Thanks, @danielabdi-noaa!  I have updated `modulefiles/build_jet_intel` and `modulefiles/build_orion_intel` so that the `nccmp` module is loaded after `NetCDF`.

**Comment by MichaelLueken** [comment]
The hera.intel fundamental WE2E have successfully ran through to completion.  Given @danielabdi-noaa approval, I have gone ahead and submitted the Jenkins CI testing for the machines.

**Comment by MichaelLueken** [comment]
All of the Jenkins CI pipeline tests have successfully passed.  Require one more approval before this work can be merged to the official repository.

**Review by chan-hoo — APPROVED** [review]

**Pull request merged**
